### ISSUE TITLE
link sent to verify an account after doing `pod trunk register <email>` results in a 404

### ISSUE THREAD

**Issue by swarup98** [CocoaPods/trunk.cocoapods.org#290]
* [x] I've read and understood the [*CONTRIBUTING* guidelines and have done my best effort to follow](https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md).

# Report

## What did you do?

> pod trunk register [email]
[!] Please verify the session by clicking the link in the verification email that has been sent to [email]

i then went to the email and clicked on the link sent to verify.  i got the below error once clicking:

![image](https://user-images.githubusercontent.com/3740915/92637698-31f7b900-f2a7-11ea-8e9b-a9518f8a5839.png)

## What did you expect to happen?

I expected that the verification link would work and be allowed to adminster the pod my team is working on.

## What happened instead?

Please see the attached image above.

### Stack

```
   CocoaPods : 1.9.3
        Ruby : ruby 2.6.0p0 (2018-12-25 revision 66547) [x86_64-darwin19]
    RubyGems : 3.0.1
        Host : Mac OS X 10.15.6 (19G2021)
       Xcode : 11.7 (11E801a)
         Git : git version 2.28.0
Ruby lib dir : /Users/swarupb/.asdf/installs/ruby/2.6.0/lib
Repositories : 
```

### Installation Source

```
Executable Path: /Users/swarupb/.asdf/installs/ruby/2.6.0/bin/pod
```

### Plugins

```
cocoapods-deintegrate : 1.0.4
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.0
cocoapods-stats       : 1.1.0
cocoapods-trunk       : 1.5.0
cocoapods-try         : 1.2.0
```

## Project that demonstrates the issue

This is an issue w/ CocoaPods on the command line.

**Comment by dnkoutso** [comment]
Not sure what the issue is here but transferred to the right repo.

**Comment by swarup98** [comment]
hello, the issue here is that when I ask to reset the session the `Spreedly` pod, the link in the email does not work.  when you say transferred to the right repo, what exactly do you mean.  the current listed owners for the pod are no longer with the company and I need to transfer ownership to myself.

**Comment by dnkoutso** [comment]
I meant I transferred your issue to https://github.com/CocoaPods/trunk.cocoapods.org/

**Comment by chrisballinger** [comment]
I am also seeing this issue for URLs like `https://trunk.cocoapods.org/sessions/verify/xxxxxxxx`: 

<img width="1217" alt="Screen Shot 2020-11-02 at 4 06 52 PM" src="https://user-images.githubusercontent.com/518687/97932716-94a88680-1d25-11eb-97f5-39d37493914a.png">

The good thing is that the registration process works, it just looks like it doesn't. After visiting the (404ing) URL, the session in the terminal is active.

**Comment by chrisballinger** [comment]
I'm not sure if it's related, but I'm seeing this for registration URLs that have been mangled by an [Outlook's Safelinks](https://docs.microsoft.com/en-us/microsoft-365/security/office-365-security/atp-safe-links?view=o365-worldwide)™ for Enterprise feature, so at first the URL looks like `https://nam01.safelinks.protection.outlook.com/blahblahblah` and then gets redirected. I haven't yet tried reproducing using an email provider that doesn't mangle the URL.

**Comment by xlith** [comment]
Yes, registration works but there is this issue. Even though registration works, we can not verify email accounts. So we can not add the user as an owner to any pod.
